style/test.css

```diff
@@ -14,3 +14,14 @@
     padding-left: 32px;
     padding-right: 32px;
 }
+
+.logger {
```

Comment:
This file is for the test page. If you create a new page/URL you should create its own css file so its easier to maintain.